html-forms/index.html

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="index.html">
+      <div>Email</div>
```

Comment:
Please wrap your titles within the form in a label tag !
Ex:  < label for="email-address" > Email </ label >